ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-release-4.16.yaml

```diff
@@ -96,6 +96,16 @@ tests:
     test:
     - ref: windows-e2e-operator-test
     workflow: ipi-aws-ovn-hybrid
+- always_run: false
+  as: vsphere-disconnected
```

Comment:
please make this the same as master job `vsphere-disconnected-e2e-operator`

Will require a `make jobs` run after to change `jobs/` config